app/src/main/java/com/huji/cse/flatfinder/MainActivity.java

```diff
@@ -62,6 +62,8 @@ public void onCompleted(JSONObject object, GraphResponse response) {
                 });
         request.setGraphPath(getString(R.string.path_of_facebook_group_with_filters));
         request.executeAsync();
+        Intent intent = new Intent(this, MapsActivity.class);
```

Comment:
I would put it onComplete, because that's the only way we can know for certain that the request from Facebook was accepted